### ISSUE TITLE
Training Steps

### ISSUE THREAD

**Issue by thetushargoyal** [ictnlp/StreamSpeech#18]
Hey @zhangshaolei1998, I was training StreamSpeech, wanted to know how many steps you trained it for? Thank you

**Comment by Gql-1** [comment]
Hello, I'd like to ask if you have already been successfully trained in other language translations?

**Comment by thetushargoyal** [comment]
@Gql-1 Yes I was able to train it for English To French! (X-En was what they provide)

**Comment by Gql-1** [comment]
> [@Gql-1](https://github.com/Gql-1) Yes I was able to train it for English To French! (X-En was what they provide)

Can you share some of the modifications made for training new language translations? For example, changes in data formats and training code.
